src/array-methods/11/task.js

```diff
@@ -0,0 +1,22 @@
+const vacationPackages = [
+    { destination: "Maldives", days: 10, cost: 2000 },
+    { destination: "Alaska", days: 5, cost: 1000 },
+    { destination: "Switzerland", days: 7, cost: 1500 }
+];
+
+let vacationPackagesWithAverageCost = vacationPackages.map(vacationPackage => ({
+    destination: vacationPackage.destination,
+    days: vacationPackage.days,
+    cost: vacationPackage.cost,
+    averageDailyCost: Math.round(vacationPackage.cost / vacationPackage.days),
+}));
+let filteredVacationPackages = vacationPackagesWithAverageCost.filter(price => price.averageDailyCost < 200)
+let sortedDestinations = filteredVacationPackages.map(vacation => ({
+    destination: vacation.destination,
+})).sort((a, b) => {
```

Comment:
```suggestion
let sortedDestinations = filteredVacationPackages.map(({destination}) => ({ destination })).sort((a, b) => {
```

Your approach here is fine. Just showing you a different perspective, of how this could be done.